russian/light-fire/brake-the-wall/i-am-the-bee.md

```diff
@@ -0,0 +1 @@
+Вы возвращаетесь обратно и осматриваете комнату, пытаясь найти хорошее укрытие.  Замечаете зеркало прислоненное к стене у окна. Непреодолимая сила тянет вас к нему. Вы подбегете к зеркалу, и ...О, нет! ...О, да! из зеркала на вас смотрит полосатая, пушистая пчела. Вы расправляете свои перламутровые крылышки и летите прочь от зефирного кошмара.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.